api/pkg/transformer/types/series/series.go

```diff
@@ -88,6 +97,70 @@ func (s *Series) Get(index int) interface{} {
 	return s.series.Elem(index).Val()
 }
 
+// func (s *Series) Append(values interface{}) {
+// 	s.series.Append(values)
+// }
+
+// func (s *Series) Concat(x Series) *Series {
+// 	concat := s.series.Concat(*x.series)
+// 	return &Series{&concat}
+// }
+
+func (s *Series) Order(reverse bool) *Series {
```

Comment:
Can you add documentation to all of the series's method? especially for the newly added one